src/Classes/MyRadio/URLUtils.php

```diff
@@ -82,6 +82,16 @@ public static function redirectWithMessage($module, $action, $message)
     }
 
     /**
+     * Redirects to another page, specified already by the caller.
+     *
+     * @param string  $URI The relative URI to redirect to.
+     */
+    public static function redirectURI($URI)
+    {
+        header('Location: '.$URI);
```

Comment:
Hmm, I think if someone wanted to go to that page anyway, then there's easier ways of doing it :p